src/test/resources/org/jenkinsci/plugins/workflow/cps/SerialFormTest/persistence/jobs/p/builds/1/build.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
nit: Personally I prefer to use ZIP files with `@LocalData` so this kind of stuff does not come up in GitHub searches, but maybe you prefer doing things this way for other reasons.

Reply:
> come up in GitHub searches

Could be an issue I suppose. I normally prefer exploded `$JENKINS_HOME` because it is easier to review, and occasionally it even makes sense to change one line of one file, which in a ZIP would just be an opaque blob.